### PR TITLE
Add new taskotron filter.

### PR DESCRIPTION
**Pull request by ralphbean**
This is a second take #58 based on @kparal's comments there.

### CODE REVIEW

File: alembic/versions/167e0e6dd4e5_make_code_path_longer.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+# revision identifiers, used by Alembic.
+revision = '167e0e6dd4e5'
+down_revision = '38c9c18d342e'
+
+from alembic import op
+import sqlalchemy as sa
+
+def upgrade():
+    # First, make this column bigger.
```
Comment (pypingou) [inline alembic/versions/167e0e6dd4e5_make_code_path_longer.py]: Only a First? :)

**Comment by pypingou** [comment]
:+1: for me

**Comment by kparal** [comment]
This seems to add a new taskotron filter with the correct 3 rules, that's ok. But shouldn't we also exclude all taskotron events from the existing _Events on packages that I own_ filter, as discussed in #58? I don't see it in the current patch.

**Comment by ralphbean** [comment]
Good catch @kparal.  That is correct.

**Comment by kparal** [comment]
Looks good to me, no guarantees :-)

**Comment by kparal** [comment]
@mkrizek wdyt?

**Comment by mkrizek** [comment]
:+1:

**Pull request merged**